include/dd/Package.hpp

```diff
@@ -1367,7 +1377,7 @@ namespace dd {
 
             computeTable.insert(xCopy, yCopy, {e.p, e.w});
 
-            if (e.w != Complex::zero && (x.w != Complex::one || y.w != Complex::one)) {
+            if (!e.w.exactlyZero() && (x.w != Complex::one || !y.w.exactlyZero())) {
```

Comment:
if there is an `exactlyZero` function, there should also be a `exactlyOne` function that replaces the second condition in this check.

Reply:
Added the requested function.